package.json

```diff
@@ -60,13 +60,15 @@
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/plugin-proposal-decorators": "^7.14.2",
     "@babel/plugin-proposal-private-methods": "^7.13.0",
+    "@babel/plugin-transform-runtime": "7.17.0",
     "@babel/preset-env": "^7.14.4",
     "@open-wc/testing": "3.0.3",
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/commit-analyzer": "^8.0.1",
     "@semantic-release/git": "^9.0.0",
     "@semantic-release/npm": "^7.1.3",
     "@semantic-release/release-notes-generator": "^9.0.2",
+    "@tignum/shpi-report": "latest",
```

Comment:
Oh, we should no use this here! pdf-generation-actions-web-component is a public library and shpi-report is a very private Tignum template. Please remove it from here 😬

Reply:
I removed the automatic generator.